### PR TITLE
fix: streaks popup to not show

### PR DESCRIPTION
**Pull request by nensidosari**
## Changes
- missed one part on not showing the modal

### Preview domain
https://fix-streak-popup.preview.app.daily.dev

### CODE REVIEW

**Review by omBratteng — APPROVED** [review]

File: packages/shared/src/components/modals/streaks/NewStreakModal.tsx [hunk 1]
```diff
@@ -58,9 +58,8 @@ export default function NewStreakModal({
       trackEvent({
         event_name: AnalyticsEvent.DismissStreaksMilestone,
       });
+      completeAction(ActionType.DisableReadingStreakMilestone);
```
Comment (rebelchris) [inline packages/shared/src/components/modals/streaks/NewStreakModal.tsx:61]: Wait we use action for this, does it work with disabling it again?
Not sure we ever clear actions?

**Review by rebelchris — APPROVED** [review]
The code is ok, just wondering about using actions for this purpose.
Why didn't we use settings for it?

File: packages/shared/src/components/modals/streaks/NewStreakModal.tsx (hunk 1 above)
Comment (nensidosari) [reply packages/shared/src/components/modals/streaks/NewStreakModal.tsx:61]: This we can never disable, this is only about the modals. The streaks in general we're using settings - u can enable disable again. Discussion here https://dailydotdev.slack.com/archives/C05TQL6HWTH/p1717146432887299

Comment (rebelchris) [reply packages/shared/src/components/modals/streaks/NewStreakModal.tsx:61]: Oh ok this one is a once you remove it never get it back thing :)

Comment (nensidosari) [reply packages/shared/src/components/modals/streaks/NewStreakModal.tsx:61]: There's also no way ui wise even if it was with settings to get this back

**Pull request merged**